roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -30,11 +30,24 @@
 
 - name: Configure exports
   template:
-    dest: /etc/exports
+    dest: /etc/exports.d/openshift-ansible.exports
     src: exports.j2
   notify:
   - restart nfs-server
 
+# TODO: Clean up old /etc/exports like this or something
+#- name: register exports
+#  command: cat /etc/exports.d/openshift-ansible.exports
+#  register: exports_out
+#- name: remove exports
+#  line_in_file:
+#    src: /etc/exports
+#    line: {{ item }}
+#    state: absent
+#  with_items: exports_out.stdout.lines
+#  when: exports_out.rc == 0
+#
```

Comment:
@sdodson @abutcher considering that we don't really consider the nfs host to be mean for production, I don't really see an issue with not cleaning up after ourselves here.